cmd/adsysd/integration_tests/testdata/PolicyUpdate/states/old-data/cache/policies/adsystestuser@example.com/policies

```diff
@@ -64,6 +64,9 @@ gpos:
             protocol://example.com/current-old/old-mount
           disabled: false
           strategy: append
+      apparmor:
+        - key: apparmor-users
+          value: users/privileged_user
```

Comment:
This is what is applied on the new GPO content, correct? So this is another case where there is no difference between old data vs new data and we don’t really know if the new config was applied, if the old one was restored from the cache and so on?

Reply:
Shold we change the value applied itself too? That way, we don’t only check the content changes BUT also that we took into account the new key?

Reply:
right, but this cache should then be confined_user too to match the value on disk?